File: 1. Array & Hashing/242. 有效的字母异位词/242. 有效的字母异位词.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        #ord("a") = 97，依此類推...
        temp1 = [0]*26
        temp2 = [0]*26
        for sub_s in s:
            temp1[ord(sub_s) - 97] += 1
        for sub_t in t:
            temp2[ord(sub_t) - 97] += 1
        return temp1 == temp2

File: Array & Hashing/242. 有效的字母异位词/242. 有效的字母异位词.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.length() != t.length()){
            return false;
        }
        vector<int> temp(26);
        //長度相同
        for(int i = 0; i < s.size(); i++){
            temp[int(s[i])-97] ++;
            temp[int(t[i])-97] --;
        }
        for(int j = 0; j < 26; j++){
            if(temp[j] != 0){
                return false;
            }
        }
        return true;
    }
};